src/lib.rs

```diff
@@ -226,3 +237,23 @@ pub mod sync;
 #[cfg(test)]
 mod testing;
 mod waker;
+
+#[macro_export]
+/// This macro is used to replace the keyword yield to
+/// avoid using nightly features when using any of the three
+/// `proc_macro_attributes` for easy generator definition.
+///
+/// # Example
+/// ```
+/// use genawaiter::yield_;
+/// async fn odds() {
+///     for n in (1..).step_by(2).take_while(|n| *n < 10) {
+///         yield_! { n }
+///     }
+/// }
+/// ```
+macro_rules! yield_ {
+    ($val:tt) => {
+        ()
```

Comment:
How about changing this to something like:

```rust
macro_rules! yield_ {
    ($val:tt) => {
        compile_error!("forgot to use attribute")
    };
}
```

That way, if someone forgets, there's no chance of the code accidentally compiling